packs/equipment/shard-of-the-third-seal.json

```diff
@@ -34,7 +34,26 @@
             }
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "TokenLight",
+                "value": {
+                    "animation": {
+                        "intensity": 4,
+                        "speed": 1,
+                        "type": "torch"
+                    },
+                    "bright": 20,
+                    "color": "#9b7337",
+                    "dim": 40,
+                    "shadows": 0.2
+                }
+            },
+            {
+                "key": "TokenEffectIcon",
+                "value": "systems/pf2e/icons/equipment/adventuring-gear/flint-and-steel.webp"
```

Comment:
Not sure if best practice here is to duplicate the asset, or just use the existing one.